docs/pbc/all/merchant-management/202407.0/marketplace/marketplace-merchant-commission-feature-overview.md

```diff
@@ -0,0 +1,187 @@
+
+|PLACEHOLDER | DESCRIPTION |
+| - | - |
+| {OPERATOR} | The relationship between category and `{CATEGORY_VALUE}`. Accepted operators: `IS NOT IN`, `IS IN`, `=`, `!=`. |
+| {CATEGORY_VALUE} | The category for defining a condition. Accepts multiple values separated by `;`. |
+
+
+Examples:
+* Apply commission to all products in the Electronics category: `category IS IN 'electronics'`
+* Apply commission to all products in the Smartphones and Smartwatches categories: `category IS IN 'smartphones';smartwatches'`.
+
+
+If a category has child categories, applying a commission to the category applies it to the child categories too. For example, applying commission to the Electronics category also apply it to the Smart Watches category. So, you might want to set up conditions using child categories or exclude some child categories.
+
+### Item price conditions
+
+To specify conditions based on item prices, the following format is used: `item-price {OPERATOR} '{PRICE_VALUE}'`
+
+|PLACEHOLDER | DESCRIPTION |
+| - | - |
+| {OPERATOR} | The relationship between item-price and `{PRICE_VALUE}`. Accepted operators: `<`, `<=`, `=`, `!=`, `>=`, `>`, `IS IN`, `IS NOT IN`. |
+| {PRICE_VALUE} | The price to check the condition against. Accepts decimal numbers and ranges. Item price is taken from the product's gross or net price depending on the price mode of the order. The product gross price is configured in the backend (including volume pricing) and does not include any discounts applied afterward in the shopping cart or checkout. |
+
+Example: `item-price > '2' AND item-price <= '10.99'`.
+
+
+
+### SKU-based conditions
+
+
+To specify conditions based on SKU, use the following format: `SKU {OPERATOR} {SKU_VALUE}`
+
+|PLACEHOLDER | DESCRIPTION |
+| - | - |
+| {OPERATOR} | The relationship between category and `{CATEGORY_VALUE}`. Accepted operators: `IS NOT IN`, `IS IN`, `=`, `!=`. |
+| {SKU_VALUE} | The SKU to check the condition against. Accepts product concrete SKUs. Accepts multiple values separated by `;`. |
+
+Example: `SKU IS IN '136_24425591;134_29759322'`
+
+### Price mode based commissions
+
+To specify conditions based on price mode, use the following format: `price-mode = ""{PRICE_MODE}""`. For example, `price-mode = ""GROSS_MODE""`.
+
+This can be useful if you want different commissions to be applied to orders in different prices modes.
+
+This will require you to change how the commission is applied to use Apply commission on the sum item total price to pay after discounts with additions. (Represented by sumPriceToPayAggregation in GLUE API). Such customizations are easy, and we already factored that into our application design. To do that, follow the integration guide. on average, changing this logic requires 2h of developer's time: (spryker-docs/docs/pbc/all/merchant-management/202407.0/marketplace/install-and-upgrade/tutorials-and-howtos/create-merchant-commission-calculator-type-plugin.md)
```

Comment:
Do you mean that specifying price mode in conditions doesn't work by default?

Reply:
Lets remove this paragraph and move it to different part of the document: "This will require you  ... calculator-type-plugin.md).

Use case when this becomes useful is in case item-price is used in the Item conditions, since item-price changes depending on the price mode of the order marketplace operator might want to create two comissions with different conditions per price mode:

E.g. 
Rule 1: item-price > 120 and price-mode = 'GROSS_MODE' 
Rule 2: item-price > 100 and price-mode = 'NET_MODE'